clients/lsp-html.el

```diff
@@ -23,19 +23,21 @@
 ;;; Code:
 
 (require 'lsp-mode)
+(require 'f)
```

Comment:
I don't think require is necessary since `lsp-mode` already has it.

Reply:
Removed it in latest commit